index.html

```diff
@@ -7,30 +7,35 @@
   <link href="build/css/style.min.css" type=text/css rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 
-  <title></title>
+  <title>InstaNews - New York Times Reader</title>
 </head>
 
 <body>
-  <img src='images/nyt-logo.svg' alt='logo' class='header' id='header'>
+  <header>
+    <img src='images/nyt-logo.svg' alt='logo' class='header' id='header'>
 
-  <p><span>Choose a section:</span></p>
+    <p>
+      <span>Choose a section:</span>
+    </p>
 
-<select class='choose'  id='choose'>
-  <option>select...</option>
-  <option>health</option>
-  <option>food</option>
-  <option>travel</option>
-  <option>opinion</option>
-</select>
-    
-<section class='news', id='news'>
+    <select class='choose' id='choose'>
+      <option>select...</option>
+      <option>health</option>
+      <option>food</option>
+      <option>travel</option>
+      <option>opinion</option>
+    </select>
+  </header>
 
-</section>
+  <section class='news' , id='news'>
 
-<footer>
+  </section>
+
+  <footer>
     <p>© Copyright 2018 Instanews</p>
-</footer>
-<script src="https://code.jquery.com/jquery-3.1.0.js"></script>
-<script src="/Users/vanessayang/Desktop/RED/red--instanews/red-instanews/js/script.js"></script>  
+  </footer>
+  <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
+  <script src="/Users/vanessayang/Desktop/RED/red--instanews/red-instanews/js/script.min.js"></script>
```

Comment:
This shouldn't be an absolute path like this. To make this path usable outside of your local environment change it to this:
`src="./build/js/script.min.js"`

It is key that we are using the production ready version of the JavaScript